File: lab1/sesion-2/ejemplos_c/conceptos_c/estructuras/comparacion_lenguajes/python/ejemplo3.py
class Persona:
    # El constructor ahora acepta argumentos para inicializar los atributos
    def __init__(self, nombre, edad):
        self.nombre = nombre
        self.edad = edad

if __name__ == "__main__":
    # Se inicializan los objetos pasando los valores al crearlos
    p1 = Persona("Luis Gomez", 30)
    p2 = Persona("Marta Peralta", 30)

    print(f"{p1.nombre} tiene {p1.edad} años")
    print(f"{p2.nombre} tiene {p2.edad} años")

File: lab1/sesion-2/ejemplos_c/conceptos_c/estructuras/comparacion_lenguajes/java/Ejemplo3.java
public class Ejemplo3 {
    public static void main(String[] args) {
        // Se inicializan los objetos usando el constructor
        Persona p1 = new Persona("Luis Gomez", 30);
        Persona p2 = new Persona("Marta Peralta", 30);

        System.out.printf("%s tiene %d años\n", p1.nombre, p1.edad);
        System.out.printf("%s tiene %d años\n", p2.nombre, p2.edad);
    }
}